flake.nix

```diff
@@ -111,7 +111,7 @@
             docgen = {
               enable = true;
               name = "docgen";
-              entry = "${pkgs.docgen}/bin/docgen";
+              entry = "make docgen";
```

Comment:
is this the git hook check or do you mean sthg else ?

Reply:
Yep. The hook appears to be working but or some reason it's deleting all the docs :thinking: